modules/resolving-cuda-oom-errors.adoc

```diff
@@ -0,0 +1,67 @@
+:_module-type: PROCEDURE
+
+[id="resolving-cuda-oom-errors-for-the-single-model-serving-platform_{context}"]
+= Resolving CUDA out-of-memory errors
+
+[role="_abstract"]
+
+In certain cases, depending on the model and hardware accelerator used, the TGIS memory auto-tuning algorithm might underestimate the amount of GPU memory needed to process long sequences. This miscalculation can lead to Compute Unified Architecture (CUDA) out-of-memory (OOM) error responses from the model server. In such cases, you must update or add additional parameters in the TGIS model-serving runtime, as described in the following procedure.
+
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the admin group (for example, {odh-admin-group}) in {openshift-platform}.
+* You have installed the single-model serving platform. For more information, see link:{odhdocshome}/serving_models#about-the-single-model-serving-platform_serving-large-models[About the single-model serving platform]
+* You have added a custom model-serving runtime for TGIS or have created a copy of the pre-installed standalone TGIS runtime. For more information, see link:{odhdocshome}/serving_models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+endif::[]
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the admin group (for example, {oai-admin-group}) in {openshift-platform}.
+* You have installed the single-model serving platform. For more information, see link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#about-the-single-model-serving-platform_serving-large-models[About the single-model serving platform]
+* You have added a custom model-serving runtime for TGIS or have created a copy of the pre-installed standalone TGIS runtime. For more information, see link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
```

Comment:
Hmmm, to me, these are more like steps in the core procedure than prerequisites?

I think I would add a conditional step (replacing the existing step 2 in the procedure) similar to the following:

```
. Based on the runtime that you used to deploy your model, perform one of the following actions:
+
* If you used the pre-installed *TGIS Standalone ServingRuntime for KServe* runtime, you must duplicate this runtime before following the remainder of this procedure. For more information about duplicating the pre-installed TGIS runtime, see.....
* If you were already using a custom TGI runtime, click the action menu (&#8942;) next to the runtime and select *Edit*.
```